haskell/servant/servant-proj.cabal

```diff
@@ -0,0 +1,24 @@
+name:                servant-proj
+version:             0.1.0.0
+-- synopsis:
+-- description:
+category:            Web
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Lib
+  build-depends:       base >= 4.7 && < 5
+                     , servant-server
+                     , wai
+                     , warp
+  default-language:    Haskell2010
+
+executable servant-proj-exe
+  hs-source-dirs:      app
+  main-is:             Main.hs
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
```

Comment:
Either here or as a CLI option to `stack` in the Dockerfile, I'd add the `-O2` option (I *think* by default stack doesn't pass them). Probably also a few other flags (`-A64m` for example, or ` -funbox-strict-fields`) will help a lot. But usually figuring out the exact parameters requires testing in a machine similar to the one you'll ultimately be using for the benchmark.